### PR TITLE
Fix as many anys as I can

### PR DESCRIPTION
**Pull request by tbezman**
At some point we need to enable the relay eslint plugin, we have a lot of outdated fragments

image

### CODE REVIEW

File: src/components/TokenHolderList/TokenHolderList.tsx [hunk 1]
```diff
@@ -30,7 +30,10 @@ function TokenHolderList({ title, tokenHoldersRef }: Props) {
     nonNullTokenHolders.sort((a, b) => {
       const usernameA = a.user.username.toLowerCase();
       const usernameB = b.user.username.toLowerCase();
+
       return (
+        // TODO(Terence): What is going on here
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
```
Comment (tbezman) [inline src/components/TokenHolderList/TokenHolderList.tsx:36]: CC @kaitoo1 we're subtracting booleans?

File: .eslintrc.cjs [hunk 2]
```diff
@@ -40,10 +40,11 @@ const rules = {
   '@typescript-eslint/switch-exhaustiveness-check': 'off',
   '@typescript-eslint/no-namespace': 'off',
   '@typescript-eslint/no-unused-vars': 'error',
+  '@typescript-eslint/no-explicit-any': 'error',
```
Comment (tbezman) [inline .eslintrc.cjs:43]: enable the life saving lint rule

File: package.json [hunk 3]
```diff
@@ -34,6 +34,7 @@
     "bufferutil": "^4.0.5",
     "ethers": "^5.6.9",
     "lodash.clonedeep": "^4.5.0",
+    "lodash.keyby": "^4.6.0",
```
Comment (tbezman) [inline package.json:37]: more legit keyby

File: package.json [hunk 4]
```diff
@@ -107,6 +109,6 @@
     "relay-compiler": "^14.1.0",
     "ts-jest": "^27.0.7",
     "ts-prune": "^0.10.3",
-    "typescript": "^4.7.4"
+    "typescript": "^4.8.2"
```
Comment (tbezman) [inline package.json:112]: bump typescript while we're at it

File: src/components/TokenHolderList/TokenHolderList.tsx (hunk 1 above)
Comment (kaitoo1) [reply src/components/TokenHolderList/TokenHolderList.tsx:36]: classic js magic operation so we can update to make it legible and typescript compliant

File: .eslintrc.cjs (hunk 2 above)
Comment (Robinnnnn) [reply .eslintrc.cjs:43]: officially joining the big leagues

File: package.json (hunk 3 above)
Comment (Robinnnnn) [reply package.json:37]: TIL

**Review by Robinnnnn — APPROVED** [review]
thank you so much ser, incredibly satisfying 🙏🙏🙏

**Pull request merged**